tests/requirements.in

```diff
@@ -0,0 +1,11 @@
+flexmock>=0.10.6
+responses>=0.9.0,<0.10.8
+pypng
+# 5.0 is the minimum version required by latest version of pytest-html. As of
+# this update, it is 3.1.1.
+pytest>=5.0
+pytest-cov
+pytest-html
+flake8
+koji
```

Comment:
Koji is a runtime requirement, no idea why it was in tests/requirements originally. Is there a reason why we wouldn't want to install it from PyPI like everything else? @MartinBasti @rcerven

Reply:
no idea, should be in requirements

Reply:
moved to the requirements